modules/base/CMakeLists.txt

```
@@ -74,6 +74,7 @@ set(HEADER_FILES
   rendering/screenspaceframebuffer.h
   rendering/screenspaceimagelocal.h
   rendering/screenspaceimageonline.h
+  rendering/screenspacetimevaryingimageonline.h
   rendering/screenspacerenderablerenderable.h
```

```suggestion
  rendering/screenspacerenderablerenderable.h
  rendering/screenspacetimevaryingimageonline.h
```

Alphabetical sorting